vendor/broccoli-fontawesome-pack.js

```diff
@@ -46,12 +47,19 @@ FontAwesomePack.prototype.build = function() {
   if(this.options.icons === 'all'){
     selectedIcons = Object.keys(pack[pack.prefix])
   } else {
-    selectedIcons = this.options.icons
+    selectedIcons = this.options.icons.map(iconName => {
+      const prefix = config.familyPrefix;
+      if (iconName.substr(0, 2) === prefix) {
+        return iconName;
+      }
+
+      return prefix + iconName.charAt(0).toUpperCase() + iconName.substr(1);
```

Comment:
(Sorry for the late reply)

Yeah I think this is going to be a problem. It looks like this is a naive camelCase'r.

So an icon named `user-secret` would turn into `faUser-secret` I think? @jrjohnson does that sound correct?

We probably need to pull in `humps` or something similar to do this conversion.